wolfcrypt/src/debug.c

```diff
@@ -27,6 +27,25 @@
 #include <wolfssl/wolfcrypt/settings.h>
 #include <wolfssl/wolfcrypt/wc_port.h>
 #include <wolfssl/wolfcrypt/types.h>
+#include <wolfssl/options.h>
+
+WOLFSSL_API const char *libwolfssl_configure_args(void) {
```

Comment:
I didn't notice this new file in your LinuxKM PR. This debug.c name is going to cause some folks naming conflict issues... Can you please rename this to wc_debug.c? Also this file needs to be added to all vcxproj files and CMakeLists.txt

Reply:
Done, renamed to wc_debug.c.

Someone already added it to CMakeLists.txt -- I've updated its name there.  TBD how to set `LIBWOLFSSL_CONFIGURE_ARGS` and `LIBWOLFSSL_GLOBAL_CFLAGS` for a `cmake` build.  With autotools they're defined in `wolfssl/options.h`, and cmake currently does this:

```
if(EXISTS "${CMAKE_CURRENT_SOURCE_DIR}/wolfssl/options.h")
    # Copy generated ./options.h
    configure_file(${CMAKE_CURRENT_SOURCE_DIR}/wolfssl/options.h 
                   ${CMAKE_CURRENT_SOURCE_DIR}/user_settings.h)
else()
   # Use template
   configure_file(${CMAKE_CURRENT_SOURCE_DIR}/wolfssl/options.h.in 
                  ${CMAKE_CURRENT_SOURCE_DIR}/user_settings.h)
endif()
```
The if-false clause would need to be refactored to add substantive definitions, but that's a puzzle for another day.  For now the behavior with a pure-`cmake` build will be that `libwolfssl_configure_args()` and `
libwolfssl_global_cflags()` will return null.